packages/writing-style/styles/commercetools/Acronyms.yml

```diff
@@ -45,6 +46,7 @@ exceptions:
   - MIT
   - NET # otherwise ".NET" is complaining but it's generally known among developers
   - NPM
+  - OIDC
```

Comment:
I found this only ones in our docs? Is it planned to be used more often?

Reply:
We use it a bunch of times in the new Custom Apps docs.

Reply:
It's also a common acronym for user authentication topics.

Reply:
I think "OpenID Connect" is not too long or unwieldy to not directly use it instead of an acronym.  In general, if the full form is easy to read and understand, always use the full form. 

If you really want "OIDC" we have to whitelist it though because vale would not detect the initial explanation on a page because it's not these four words)

Reply:
sorry for the unclear text:   I vote for not using "OIDC" but spelling it out "OpenID Connect".  That is common practice on other documentation sites I remember (just spent a few hours on the topic in the self learning project two days ago)

Reply:
Ok, I'm fine with settling for that. Thanks for the feedback.